workflow/rules/annotation/pass2.smk

```diff
@@ -18,33 +19,37 @@ def get_annotation_vcfs(idx=False):
 
 #What about multiple ID Fields?
 rule annotate_vcfs:
+    threads:
+        4
     input:
-        bcf="calls/{prefix}.bcf",
+        bcf="results/calls/{prefix}.bcf",
         annotations=get_annotation_vcfs(),
         idx=get_annotation_vcfs(idx=True)
     output:
-        "calls/{prefix}.db-annotated.bcf"
+        "results/calls/{prefix}.db-annotated.bcf"
     params:
         extra="-Xmx4g",
         pipes=get_annotation_pipes
     conda:
         "../../envs/snpsift.yaml"
     shell:
-        "bcftools view {input.bcf} {params.pipes} | bcftools view -Ob > {output}"
+        "bcftools view --threads {threads} {input.bcf} {params.pipes} | bcftools view --threads {threads} -Ob > {output}"
 
 
 rule annotate_dgidb:
+    threads:
+        4
     input:
-        "calls/{prefix}.bcf"
+        "results/calls/{prefix}.bcf"
     output:
-        "calls/{prefix}.dgidb.bcf"
+        "results/calls/{prefix}.dgidb.bcf"
     params:
     conda:
         "../../envs/annotate_dgidb.yaml"
     resources:
         dgidb_requests=1
     shell:
-        "rbt vcf-annotate-dgidb {input} > {output}"
+        "rbt vcf-annotate-dgidb -g 500 {input} |  bcftools view --threads {threads} -Ob > {output}"
```

Comment:
Explicitly setting genes to 500 is not required as this is already the default in the latest rbt version.